src/components/SetUp/FirstForm.jsx

```diff
@@ -0,0 +1,115 @@
+import { useState } from 'react';
+import {
+  Box,
+  Button,
+  FormControl,
+  FormLabel,
+  Input,
+  VStack,
+  Heading,
+  Text,
+  Link,
+  HStack,
+  Image,
+  Flex,
+} from '@chakra-ui/react';
+import { useNavigate } from 'react-router-dom';
+import LOGO from './fph_logo.png';
+import { useAuth } from '../../contexts/AuthContext';
+import PropTypes from 'prop-types'; // Import PropTypes
+
+const FirstForm = ({ isAdmin, nextStep }) => {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [confirmPassword, setConfirmPassword] = useState('');
+  const { signup } = useAuth();
+  const [error, setError] = useState('');
```

Comment:
🚫 **[eslint]** <[no-unused-vars](https://eslint.org/docs/latest/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'error' is assigned a value but never used.